lgsm/functions/info_game.sh

```diff
@@ -93,6 +93,31 @@ fn_info_game_arma3(){
 	battleeyeport=$((port+4))
 }
 
+fn_info_game_armareforger(){
+	# Config
+	if [ ! -f "${servercfgfullpath}" ]; then
+		servername="${unavailable}"
+		adminpassword="${unavailable}"
+		serverpassword="${unavailable}"
+		maxplayers="${zero}"
+	else
+		servername=$(grep "name" "${servercfgfullpath}" | sed -e 's/^[ \t]*//g' -e '/^\//d' -e 's/name//g' | tr -d '=\";,:' | sed -e 's/^[ \t]*//' -e 's/[ \t]*$//')
```

Comment:
Like @Marcin648 i would prefer here the parsing via jq - this is something I would not do anymore for configparsing.
Note: also the configip should be added as well.
See this parsing as example this:
https://github.com/h3o66/LinuxGSM/commit/0757585ea3b8a8361e62895fa265f4a7b43f0f55#diff-670814a6c9341dd77e070ca74832863054f3e3998d5a339606b744e7bb3480f9